### DISCUSSION TITLE
Streamed video not playing on video players only playing on a browser

### DISCUSSION THREAD

**Discussion by Nkwebana** [fent/node-ytdl-core#1014] [Q&A]
Good day,

I cloned and ran your solution and it works thank you, but when I play the download or streamed file it does not play on video players and it only play on google Chrome browser. I also tried to convert it to mp4 but still it does not play. Could  you please assist by giving an advice on how I can make it play?

**Comment by TimeForANinja** [comment]
* try sth like `ffprobe` to check what media / container format you're actually dealing with
* update the file-ending accordingly
* try playing it again
